pkg/operator/ceph/disruption/clusterdisruption/location.go

```diff
@@ -194,7 +194,7 @@ func getOSDsForNodes(osdDataList []OsdData, nodeList []*corev1.Node, failureDoma
 }
 
 func getFailureDomainMapForOsds(osdDataList []OsdData, failureDomainType string) (map[string][]OsdData, error) {
-	failureDomainMap := make(map[string][]OsdData, 0)
+	failureDomainMap := make(map[string][]OsdData)
 	unfoundOSDs := make([]string, 0)
```

Comment:
It didn't complain about this one too?
```suggestion
	unfoundOSDs := make([]string)
```

Reply:
while making a slice of string we don't need to give length not needed in the case of the map.